tests/integration_tests/test_botorch.py

```diff
@@ -78,9 +79,9 @@ def candidates_func(
         train_obj: torch.Tensor,
         train_con: Optional[torch.Tensor],
         bounds: torch.Tensor,
-    ) -> torch.Tensor:
+    ) -> Any:
         # Must be a `torch.Tensor`, not a list.
-        return torch.rand(1).tolist()
+        return [torch.rand(1).tolist()]
```

Comment:
In `checks` CI job, `torch` is not installed, and `mypy` cannot infer the type of this return value.
IMO, we can simply return a list of immediate values here. Or, do we have reason to stick `torch.rand`?

Reply:
Good point, I think `[0.0]` would suffice.

Reply:
Could you fix the line? Or, is it out of scope of this PR?

Reply:
There are more than one places where `torch.rand` is used, and we might as well replace them all in another PR.

Reply:
I think we can replace the line with `[0.0]`. As a result, we can create an issue with `contribution welcome and` good-first-issue`.

Reply:
@nzw0301 Do you mean that I should fix this in this PR?

Reply:
Sorry, I meant that this PR can fix it.

Reply:
Fixed.

Reply:
thanks!